data-esnext.js

```diff
@@ -1431,7 +1434,7 @@ exports.tests = [
     babel7corejs2: true,
     babel7corejs3: true,
     chrome1: false,
-    chrome91: true,
+    chrome92: false,
```

Comment:
This one was added mistakenly probably

Reply:
This code returns `true` for me in Chromium 90 :thinking: 

```js
(() => {
      let ok = false;
    class A {
      static { ok = true; }
    }
    return ok;
 })();
```

Reply:
Oh I had the "Experimental JavaScript" flag enabled.

Reply:
Thanks, I will add it under flag

Reply:
@nicolo-ribaudo are you sure it's Chrome 90? I see Chrome 91 mentioned [here](https://v8.dev/features/class-static-initializer-blocks). I have only Chrome 92 installed on this machine, so can't check

Reply:
It was 91 in https://github.com/kangax/compat-table/pull/1716, so I'm going to merge current version for now, we can change to 90 later if necessary

Reply:
I use Brave (which is Chromium-based), and the `chrome://version` page says `Chromium: 90.0.4430.212` and `V8 9.0.257.29`.

Reply:
It's not available in Chrome 92 to 94 by default